core/src/main/java/discord4j/core/spec/MessageCreateSpec.java

```diff
@@ -57,17 +56,154 @@
     private AllowedMentionsData allowedMentionsData;
     private MessageReferenceData messageReferenceData;
 
+
+    // Constants
+    public final static byte MARKDOWN_ITALIC = 0;
+    public final static byte MARKDOWN_BOLD = 1;
+    public final static byte MARKDOWN_STRIKETHROUGH = 2;
+    public final static byte MARKDOWN_UNDERLINE = 3;
+    public final static byte MARKDOWN_CODELINE = 4;
+
     /**
-     * Sets the created {@link Message} contents, up to 2000 characters.
+     * Sets the specs contents, up to 2000 characters.
      *
      * @param content The message contents.
      * @return This spec.
      */
     public MessageCreateSpec setContent(String content) {
-        this.content = content;
+        this.contentBuilder.setLength(0);
```

Comment:
To prevent code duplication this should callback to the new `#resetContent` method.